### PR TITLE
Add Architecture Overview page

### PR DESCRIPTION
**Pull request by aimichelle**
This page adds a breakdown of each component in Pixie (Vizier, Cloud, Operator). 
It does not contain any diagrams yet.

Signed-off-by: Michelle Nguyen <michellenguyen@pixielabs.ai>

### CODE REVIEW

File: content/en/01-about-pixie/03-pixie-ebpf.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "How Pixie uses eBPF"
-metaTitle: "About Pixie | How Pixie uses eBPF"
+metaTitle: "About Pixie | Architecture | How Pixie uses eBPF"
 metaDescription: "How Pixie uses eBPF to automatically collect data"
-order: 3
+order: 2
```
Comment (htroisi) [inline content/en/01-about-pixie/03-pixie-ebpf.md]:
```suggestion
order: 3
```

File: content/en/01-about-pixie/03-pixie-ebpf.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "How Pixie uses eBPF"
-metaTitle: "About Pixie | How Pixie uses eBPF"
+metaTitle: "About Pixie | Architecture | How Pixie uses eBPF"
```
Comment (htroisi) [inline content/en/01-about-pixie/03-pixie-ebpf.md]:
```suggestion
metaTitle: "About Pixie | How Pixie uses eBPF"
```

**Review by htroisi — DISMISSED** [review]

File: content/en/01-about-pixie/03-pixie-ebpf.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 title: "How Pixie uses eBPF"
 metaTitle: "About Pixie | How Pixie uses eBPF"
 metaDescription: "How Pixie uses eBPF to automatically collect data"
-order: 3
+order: 3
```
Comment (philkuz) [inline content/en/01-about-pixie/03-pixie-ebpf.md]:
```suggestion
order: 3
```

Comment (philkuz) [reply content/en/01-about-pixie/03-pixie-ebpf.md]: unnecessary trailing space and file change

File: content/en/05-reference/02-architecture.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+- **Vizier Operator**: Pixie's [Kubernetes operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) which is used to help deploy and manage Vizier on a cluster, such as keeping all components running and up-to-date. 
+
+## Vizier
+
+Vizier is Pixie's data plane. It is responsible for collecting and processing data within the cluster that is being monitored.
+
+### Components
+
+1. **Pixie Edge Module (PEM)**: The data collectors, deployed to each node via a daemonset. PEMs serve as a short-term in-memory store (up to 24h) for all data collected in the cluster. The PEMs are also responsible for processing this data when they receive a script execution request. 
+2. **Collector (Kelvin)**: The Kelvin is the higher-level data collector, where PEMs send data to for further aggregation (e.g. joins, unions)  and processing.
```
Comment (philkuz) [inline content/en/05-reference/02-architecture.md]:
```suggestion
2. **Collector (Kelvin)**: Kelvin is the high-level data collector. During script execution, PEMs send partially processed data to Kelvins. Kelvins aggregate the node level view from each PEM into a cluster-level view and complete the script execution steps that requires all of the data, such as aggregates and joins.
```

File: content/en/05-reference/02-architecture.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+## Vizier
+
+Vizier is Pixie's data plane. It is responsible for collecting and processing data within the cluster that is being monitored.
+
+### Components
+
+1. **Pixie Edge Module (PEM)**: The data collectors, deployed to each node via a daemonset. PEMs serve as a short-term in-memory store (up to 24h) for all data collected in the cluster. The PEMs are also responsible for processing this data when they receive a script execution request. 
+2. **Collector (Kelvin)**: The Kelvin is the higher-level data collector, where PEMs send data to for further aggregation (e.g. joins, unions)  and processing. 
+3. **Cloud Connector**: The cloud connector communicates with Pixie Cloud and passes messages between the two systems. These messages range from  heartbeats, used to indicate the Vizier's overall status, and script execution requests and results.
```
Comment (philkuz) [inline content/en/05-reference/02-architecture.md]:
```suggestion
3. **Cloud Connector**: The cloud connector handles message passing between Vizier and Pixie Cloud. These messages range from  heartbeats, used to indicate the Vizier's overall status, and script execution requests and results.
```

File: content/en/05-reference/02-architecture.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+4. **Metadata**: The metadata service serves as the hub for all of Vizier's metadata. This includes K8s metadata, used to contextualize any collected data, and the status of each PEM running in the cluster.
```
Comment (philkuz) [inline content/en/05-reference/02-architecture.md]:
```suggestion
4. **Metadata**: The metadata service serves as the hub for all of Vizier's metadata. The metadata stored includes k8s objects (ie pods, services, namespaces) for adding context to execute scripts as well as metadata about the Vizier state (ie PEM status, [dynamic logs deployed](https://docs.px.dev/tutorials/custom-data/dynamic-go-logging/)) 
```

**Review by philkuz — COMMENTED** [review]
Check out my suggested changes.

**Review by htroisi — DISMISSED** [review]

**Review by htroisi — APPROVED** [review]

**Pull request merged**